lld/test/COFF/gc-dwarf-eh.s

```diff
@@ -1,5 +1,10 @@
 # REQUIRES: x86
 
+# Swift LLVM fork downstream change start
+# Disable failing test
+# XFAIL: *
+# Swift LLVM fork downstream change end
+
```

Comment:
Why are we disabling the `gc-dwarf-eh.s` test for COFF here? (That isn't anything to do with Mach-O).

Reply:
Removed this change